sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/Models/ConversationsModelFactory.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Azure.AI.Language.Conversations.Models
+{
+    /// <summary> Model factory for read-only models. </summary>
+    public static partial class ConversationsModelFactory
```

Comment:
The code generator should create this automatically for you. Was there some reason you defined it yourself?

Reply:
It was removed from the model factory when I regenerated with changes in the `Result` object. Not sure why it did that, but I thought it was important so I added it myself.